site.yml

```diff
@@ -21,6 +21,34 @@
       delegate_to: localhost
       changed_when: false
 
+    - name: Create folder to store config
+      file: path={{ jenkins_job_config_file_src }} state=directory mode=0755 owner=jenkins group=jenkins
```

Comment:
Can you break this out into the expanded YAML form instead of the single line form? That's the syntax I prefer as it should be simpler to add validation and lint testing in the future.